.github/workflows/ci_build_android.yml

```diff
@@ -0,0 +1,50 @@
+name: Build Android library
+
+on:
+  workflow_call:
+    outputs:
+      artifact-dir-base:
+        description: The base name of artifact directory (without the target suffix)
+        value: ${{ jobs.build-android-libs.outputs.artifact-dir-base }}
+
+jobs:
+  build-android-libs:
+    runs-on: ubuntu-20.04
+    timeout-minutes: 35
+    strategy:
+      matrix:
+        target: ["arm64-v8a", "x86_64"]
+    env:
+      ARTIFACT_DIR_BASE: ${{ github.job }}
+    outputs:
+      artifact-dir-base: ${{ env.ARTIFACT_DIR_BASE }}
+    steps:
+      - uses: actions/checkout@ec3a7ce113134d7a93b817d10a8272cb61118579 # v2.4.0
+
+      - name: Setup the rust CI.
+        uses: ./.github/actions/setup-job
+        with:
+          rust: true
+          dart: true
+          android: true
+          rust-target: "aarch64-linux-android x86_64-linux-android"
+          github_token: ${{ secrets.GITHUB_TOKEN }}
+
+      - name: Build Android library
+        run: |
+          export PATH="$(pwd)/cargo-installs/bin:${PATH}"
+          just compile-android "${{ matrix.target }}"
```

Comment:
```suggestion
          just compile-android ${{ matrix.target }}
```

Reply:
I think the general advice that shell linters (e.g. shellcheck) give is to quote variables / dynamic values at all times, to avoid unpleasant surprises. While this may not be needed here, I mostly try to follow that pattern :)

Reply:
you normally need quotes if the variable can be empty, contain spaces or wildcards. but I'm ok if you leave the quotes here